student.rb

```diff
@@ -0,0 +1,17 @@
+require './person'
+
+class Student < Person
+  def initialize(age, name = 'unknown', classroom:, parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\(ツ)/¯'
+  end
+
+  def add_classroom(classroom)
+    @classroom.student = classroom
```

Comment:
Kindly remove `.student`, and add an `attribute reader` for the `@classroom` instance variable at the top of the class.